low-level/files/opt/seagate/sspl/setup/resource_map/storage.py

```diff
@@ -158,6 +159,42 @@ def get_psu_info(self):
             data.append(psu_dict)
         return data
 
+    def get_platform_sensors(self):
```

Comment:
Suggestion on method name.
`get_platform_sensors_info` like other fru method names as we are returning respective fru data only.

Reply:
Done.